src/rws_client.cpp

```diff
@@ -622,6 +644,11 @@ std::string RWSClient::getLogText(const bool verbose)
   return ss.str();
 }
 
+std::string RWSClient::generateConfigurationPath(const std::string& topic, const std::string& type)
+{
+  return Resources::RW_CFG + "/" + topic + "/" + type;
```

Comment:
Should we be using [Poco.URI](https://pocoproject.org/docs/Poco.URI.html) here?

And elsewhere we are dealing with/constructing URIs?

Reply:
That's a good idea, I will open an issue for it.

Reply:
I have opened issue https://github.com/ros-industrial/abb_librws/issues/48 for this.